### DISCUSSION TITLE
Is the value payload of a user variable limited to 56 characters?

### DISCUSSION THREAD

**Discussion by cmaahs** [wezterm/wezterm#3256] [Q&A]
﻿In this receiver: 

`wezterm.on('user-var-changed', function(window, pane, name, value)`

Is the `value` limited to 56 characters?

**Comment by wez** [comment]
There are no limits on the length of the name or the value enforced by wezterm for user vars

**Reply by cmaahs** [reply]
Thanks much, it was apparently the example I was following, I split out the base64 of the data to a separate variable and things went much better!

thanks for all the fun...

**Reply by nekowinston** [reply]
I'm assuming that was `echo "..." | base64` vs `echo "..." | base64 -w0`? Just to clarify if anyone else finds this Q&A.
